src/receiver.rs

```diff
@@ -195,6 +200,11 @@ fn receive_packets<T>(recv: PacketReceiver, kill: Arc<AtomicBool>, send: Sender<
         trace!("Waiting on packet at {}...", recv);
         let (msg_bytes, addr) = match recv.recv_pckt() {
             Ok((bytes, addr)) => (bytes, addr),
+            Err(ref err) if err.kind() == io::ErrorKind::WouldBlock => {
```

Comment:
While testing on Windows I found that timeouts do not return ```WouldBlock``` but instead ```TimedOut```. Is this behavior consistent on Mac OS as well?